src/main/java/Task.java

```diff
@@ -0,0 +1,23 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString() {
+        return "[T]" + (isDone ? "[X]"+ this.description : "[ ]"+ this.description); // mark done task with X
```

Comment:
Could add new lines to improve readability, and comment could be incorporated as part of javadocs

```suggestion
/*
 * @return (what it returns)
 */
        return "[T]" 
                + ( (isDone ? "[X]"+ this.description) : "[ ]"+ this.description); 
```